Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java

```diff
@@ -0,0 +1,130 @@
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListView;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import nl.soccar.library.Game;
+import nl.soccar.library.Player;
+import nl.soccar.library.Room;
+import nl.soccar.library.Session;
+import nl.soccar.library.SessionData;
+import nl.soccar.ui.rmi.ClientController;
+import nl.soccar.ui.Main;
+import nl.soccar.ui.fx.FXMLConstants;
+
+/**
+ * FXML Controller class
+ *
+ * @author PTS34A
+ */
+public class GameResultsFXMLController implements Initializable {
+
+    @FXML
+    private Label lblRoomName;
+    @FXML
+    private Label lblScoreEnd;
+    @FXML
+    private TableColumn tbclBlueGoalsScored;
+    @FXML
+    private TableColumn tbclBlueUsername;
+    @FXML
+    private TableColumn tbclRedGoalsScored;
+    @FXML
+    private TableColumn tbclRedUsername;
+    @FXML
+    private Button btnExitResults;
+    @FXML
+    private Button btnLogOut;
+    @FXML
+    private Label lblUsername;
+    @FXML
+    private Label lblCar;
+    @FXML
+    private TableView<ResultTableItem> tblBlueGoalsList;
+    @FXML
+    private TableView<ResultTableItem> tblRedGoalsList;
+
+    private static final Logger LOGGER = Logger.getLogger(GameResultsFXMLController.class.getSimpleName());
+
+    private Session currentSession;
+
+    private Player currentPlayer;
+
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        ClientController controller = ClientController.getInstance();
+        currentPlayer = controller.getCurrentPlayer();
+        currentSession = currentPlayer.getCurrentSession(); // Will never be null.
+
+        
+        btnLogOut.setOnAction(e -> Main.getInstance().logOut());
+        btnExitResults.setOnAction(e -> closeResults());
+        
+        tbclBlueUsername.setCellValueFactory(new PropertyValueFactory<>("username"));
+        tbclRedUsername.setCellValueFactory(new PropertyValueFactory<>("username"));
+        tbclBlueGoalsScored.setCellValueFactory(new PropertyValueFactory<>("goalsScored"));
+        tbclRedGoalsScored.setCellValueFactory(new PropertyValueFactory<>("goalsScored"));
+
+        lblUsername.setText(currentPlayer.getUsername());
+        lblCar.setText(currentPlayer.getCarType().toString());
+
+        setGameInfo();
+
+        Optional<SessionData> session = controller.getAllSessions().stream().filter(s -> s.getRoomName().equals(lblRoomName.getText())).findFirst();
+        if (!session.isPresent()) {
+            LOGGER.log(Level.WARNING, "An exception occured while getting the SessionData from the Game Server");
+            return;
+        }
+    }
+
+    /**
+     * Method that display the current settings of the room on the session view.
+     */
+    public void setGameInfo() {
+        Room room = currentSession.getRoom();
+        Game game = currentSession.getGame();

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "game" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "game". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1854)